packages/harpguru-core/src/utils/get-menu-styles-and-animation-vals/get-menu-styles-and-animation-vals.ts

```diff
@@ -131,12 +131,12 @@ export const getMenuStylesAndAnimationVals = (
     inputRange: [0, 1],
     outputRange: [colors.pageColor, colors.homeRowsColor],
   })
-
-  // Label animation values
-  const labelOpacity = interpolate(hideMenuVal, {
+  const menuOpacity = interpolate(hideMenuVal, {
     inputRange: [0, 1],
-    outputRange: [0, 1],
+    outputRange: [0.9, 1],
```

Comment:
If the `overlayOpacity` global style variable is created, this is another place for it to be used.

Reply:
Done